src/logic/calculate.js

```diff
@@ -78,10 +87,20 @@ export default function calculate(obj, buttonName) {
     return {};
   }
 
+  // Button must be an operation
+
+  // When the user presses an operation button without having entered
+  // a number first, do nothing.
+  // if (!obj.next && !obj.total) {
+  //   return {};
+  // }
+
+  // User pressed an operation after pressing '='
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused lines of code in order for us to have a clean code-base.